src/components/dashboard/users/UserPasswordForm.tsx

```diff
@@ -28,17 +30,17 @@ const UserPasswordForm: FunctionComponent<IProps> = ({ userId }: IProps) => {
             return;
         }
 
-        const user = await window.ApiClient.getUser(userId);
+        user.current = await window.ApiClient.getUser(userId);
```

Comment:
What is the advantage to using a ref for this value?

Reply:
Sorry I'm still new to React 😅

I was unable to use a state within the onSubmit() function. It always threw an error. So I thought that a ref would be the next best thing.

Reply:
Using `state` _might_ be better, but that would require a larger refactor of this form... so I think this is fine.